.github/workflows/ci.yml

```diff
@@ -22,10 +22,10 @@ jobs:
           components: rustfmt, clippy
       - uses: Swatinem/rust-cache@v2
       - name: cargo fmt
-        run: cargo fmt -- --check
+        run: cargo fmt --all --check
```

Comment:
If I look at 
```
cargo check --help
```
it says that `--all` is deprecated
```
Package Selection:
      --all               Alias for --workspace (deprecated)
```

Maybe switch to `--workspace`?